lib/api/contacts.js

```diff
@@ -3,3 +3,73 @@ export const queryEmails = (data) => ({
     method: 'get',
     data
```

Comment:
use `params` here.

Reply:
Actually those first 5 lines of code were already there before I started touching the file. Actually there's a GET call, `queryContactEmails` with the same route later in the file. So I guess maybe someone started the job but didn't finish it. I'll erase the lines